### ISSUE TITLE
Need Animations

### ISSUE THREAD

**Issue by hiruthicShaSS** [hiruthic2002/pLANTEr#1]
Need animations in the website:
Animations should consist of:

- [ ] Growing trees or plants
- [ ] Forests
- [ ] Greenish

And put in your craziness, we expect some craziness in every **PR**😁

**Comment by aaronzshey** [comment]
Would GIFs be ok?  Or I can add organic Paper.js animations.
I just forked it, give me the greenlight and I'll start coding :+1:

**Comment by hiruthicShaSS** [comment]
I am not sure of the GIFs as it cannot be controlled manually for our needs. I just looked up Paper.js, I think it looks fun so if you can do that it would be nice because it has mouse and keyboard interaction. 
Happy coding😀.

**Comment by aaronzshey** [comment]
Awesome!  Do you have anything specific in mind @hiruthic2002 ?

**Comment by hiruthicShaSS** [comment]
Yeah, The API will be returning the data(Array) containing [_Total Stars, Total Commits, Total PRs, Total Issues_]. So I am thinking of some growing trees or forests something like that according to the number of commits and PRs.

Eg, Let's say we got 90 commits and 30 PR's then we can animate 30 trees growing up and covering the page. The calculations are rough but you get the idea.

**If you have any idea of better way to show animations with respect to the people work on OpenSource, please go ahead and work with that too and show us**

**Comment by aaronzshey** [comment]
Hmmm... interesting.  Unfortunately Paper.js can't work with external information - it exists as its own "PaperScript".  I was thinking of adding a small attention-getter on the background, like if you click, flowers would emanate from your cursor.  Do you have an example of the API's output?

**Comment by aaronzshey** [comment]
Also make sure to hacktoberfest tag to this issue and when you accept my PR, hacktoberfest-accepted to my PR - 
<img width="428" alt="Screen Shot 2020-10-05 at 10 48 23 AM" src="https://user-images.githubusercontent.com/44917034/95114119-4c7f4f80-06f8-11eb-8ff4-ac6876d5d2c8.png">


<img width="680" alt="Screen Shot 2020-10-05 at 10 48 42 AM" src="https://user-images.githubusercontent.com/44917034/95114146-586b1180-06f8-11eb-95d7-9bbb8b4c5ff3.png">

😅

**Comment by aaronzshey** [comment]
[Here's an example of what I can add to the background](http://sketch.paperjs.org/#V/0.12.7/S/nZRRi5tAEMe/yiAUtLFbyVEoSe3LHX3qQaGPSR72dIzbmJ2wu+qlR757Z9VEPa5QaiCrs///7G/GSV4CLY8YrIKfB3RZGcRBRrl/bqQBWSlpIYWToV+YOSEzpxr8Ls9oRFaqKjeot/wpas07pCEzKB3eK5NVGGaoHZoYjMwjeNlq4MtnzbptTquxhR/SlWIwDJretxrWuA9yElXblV+HiHWGDnhPFbG2W4SROqdjGG31JVp7Dd9s9Qxq8zmJ4VOyi+EuYZ3n0fWRin7bF7v0FTUKW0H6kWqLD9Rqjl+LDLFhsGtFBZnQJ1GsSNa8fJnl48hi0WvnGF0ScSLF388xTB/PDPfo23ItB94zsS+qr2fSb9LfDL++OdJbTN2bFBXqvSsnTMPV7W7UjgGs8onFGRbpGH6iWudWtCp3JXyEZTJaVQHhTZdLJ4UtVeEwhzQFZ2rkgzLSTuka15fRh5XFzvzATRGaWi7zHSy9K5mxvc33DB/SGcdMNufoMEbl5S9n3/372Yv/PntiulG8brOjE3yFbgKt+o2iRLUv3Wuwq8tTsw0LpTGfKbjtlioUFe3DbWDwSA0rgugm8vPU3/Fc+R/KxNCnT8ayIz94/PfwxGN86AbVBqvN7vIH) 

I could also make them leaves or such if you have an y SVG files.

**Comment by aaronzshey** [comment]
#6

**Comment by hiruthicShaSS** [comment]
> Hmmm... interesting. Unfortunately Paper.js can't work with external information - it exists as its own "PaperScript". I was thinking of adding a small attention-getter on the background, like if you click, flowers would emanate from your cursor. Do you have an example of the API's output?

Output:
{"data":["18", "2020", "116", "22", "7", "11"],"info":{},"time":1.5307989120483398}

Index of the data key:
1 - Total Stars
2 - Current Year
3 - Total Commits
4 - Total PR's
5 - Total Issues
6 - Contributed to

**Comment by hiruthicShaSS** [comment]
> #6

That's good, Can you attach a screenshot of how the website has changed due to your commits!

And kindly please answer these:
- [ ] My change will not break the existing code
- [ ] I have added comments for future developers

**Comment by hiruthicShaSS** [comment]
> Also make sure to hacktoberfest tag to this issue and when you accept my PR, hacktoberfest-accepted to my PR -
> <img alt="Screen Shot 2020-10-05 at 10 48 23 AM" width="428" src="https://user-images.githubusercontent.com/44917034/95114119-4c7f4f80-06f8-11eb-8ff4-ac6876d5d2c8.png">
> 
> <img alt="Screen Shot 2020-10-05 at 10 48 42 AM" width="680" src="https://user-images.githubusercontent.com/44917034/95114146-586b1180-06f8-11eb-95d7-9bbb8b4c5ff3.png">
> 
> 😅

Hey, thank you for informing... I have added "hacktoberfest" tag to the repo's topic and added "hacktoberfest-accepted" label in the specific issues too, Confirm it with the new changes and ping me if I have made any mistakes.

**Comment by aaronzshey** [comment]
The website hasn't changed at all because of my commits 😅 - did you see the Paper.JS sketch?  I'm thinking about making that the background, except replacing the circles with flowers or leaves or something.

Thanks for the API response snippet!  I'll mock a gradient throbber and hopefully make a commit tomorrow.

**Comment by hiruthicShaSS** [comment]
> The website hasn't changed at all because of my commits 😅 - did you see the Paper.JS sketch? I'm thinking about making that the background, except replacing the circles with flowers or leaves or something.
> 
> Thanks for the API response snippet! I'll mock a gradient throbber and hopefully make a commit tomorrow.

Yeah, I saw the Paper.JS sketch but when I tried to reproduce your commits on my local machine I didn't see any change on the site that's why I ask an screenshot.
##### And also the animations were awesome I'd really like to have those animations on this project. ❤

**Comment by aaronzshey** [comment]
Awesome!  I'll work on this tomorrow morning.  
Also, the issues should have the tag "hacktoberfest" - "hacktoberfest-accepted" is only for pull requests.  
Like this: (doesn't have to be pink:)
<img width="113" alt="Screen Shot 2020-10-05 at 8 51 48 PM" src="https://user-images.githubusercontent.com/44917034/95156937-972ab700-074c-11eb-96df-a15c21859497.png">

**Comment by aaronzshey** [comment]
#6 
- [x] My change will not break the existing code (no changes were made to existing code).
- [x] I have added comments for future developers

**Comment by aaronzshey** [comment]
#6  closed.  Refer to #9   

Also , could you check in all the boxes in the original post?

**Comment by aaronzshey** [comment]
Now that #9  is closed you can probably move this to "Done" in the To Dos.

**Comment by hiruthicShaSS** [comment]
Hmm, let it stay for some days and lets close it after hactoberfest completes. It will be usefull for some new contributors to make some small PR's

**Comment by aaronzshey** [comment]
:+1:
